common/core/desktop/src/kmx/kmx_processor.cpp

```diff
@@ -242,6 +243,8 @@ KMX_BOOL KMX_Processor::ProcessGroup(LPGROUP gp, KMX_BOOL *pOutputKeystroke)
 
   assert(kkp != NULL);
 
+  m_miniContextIfLen = xstrlen(kkp->dpContext) - xstrlen_ignoreifopt(kkp->dpContext);
```

Comment:
Find the number of `if()` statements in the string; assumption is `if()` statements will be at the start of the context (this should really be validated in the compiler).